gestion_base_adresse/processing/data/data_parcelle.py

```diff
@@ -225,8 +225,9 @@ def processAlgorithm(self, parameters, context, feedback):
 
         sql = """
             UPDATE adresse.point_adresse pa
-            SET id_parcelle = (SELECT p.fid FROM adresse.parcelle p
-            WHERE ST_intersects(pa.geom, p.geom));
+            SET id_parcelle = p.fid
+            FROM adresse.parcelle p
+            WHERE ST_Contains(p.geom, pa.geom)
```

Comment:
```suggestion
            WHERE ST_Contains(p.geom, pa.geom) LIMIT 1
```
Même si a priori les parcelles ne se superposent pas, on ne sait jamais.

Reply:
On ne peut pas ajouter un `LIMIT` sur ce genre de requête. Ce n'est pas une sous-requête.
J'avais testé hier justement dans mon commentaire. PostGIS prend le dernier polygone.

Reply:
Désolé j'avais mal lu la différence de code.